### PR TITLE
add information about examples

### LINKED ISSUE #23: Improve benchmark section in documentation
It is a good idea to include in the benchmark section how to run the models that we used to create the benchmark and where they are.

So, my suggestions are: 
* Explain where the models are in the documentation. 
*  Create a README file, which explains how to run and create the models in each models  folder.

### PR DESCRIPTION
**Pull request by rafaelmds**
Improve documentation to better present the examples and how to run them.

Fixes #23

### CODE REVIEW

**Comment by rafaelmds** [comment]
Currently we mention some example in section [3.4. Examples](https://ggciag.github.io/mandyoc/files/installation.html#examples).

I think we should remove this subsection and link to the benchmark or examples sections.

Also, currently there's only examples. Shall we better differentiate benchmark/example?

**Review by aguspesce — APPROVED** [review]
@rafaelmds ... Thank  you for your work! It is OK for me

**Comment by aguspesce** [comment]
@rafaelmds ... for now, you only add information about benchmark.
Are you planning to add the section on the examples?

**Comment by rafaelmds** [comment]
I'm happy with it!
Thanks again @aguspesce

**Comment by rafaelmds** [comment]
So, I think we need to move [3.4. Examples](https://ggciag.github.io/mandyoc/files/installation.html#examples) to a dedicated section (it will be section 7 after 6 Benchmarks). Then update [Benchmarks](https://ggciag.github.io/mandyoc/files/benchmarks.html#benchmarks) to reflect the new structure.

You suggested to

> create a README file, which explains how to run and create the models in each models folder.

which I think is covered by your PR #25 (for example this [README.md](https://github.com/ggciag/mandyoc/blob/f3d3e4f254d5b29a323b2242b74f951ef248e5dc/examples/Crameri2012_case2/README.md))

File: docs/files/benchmarks.rst [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+The scripts to build and run these numerical experiments are located in the `examples <https://github.com/ggciag/mandyoc/tree/main/examples>`_ folder of the Mandyoc repository.
+
+.. how-to-run-examples:
+
+**How to run examples:**
+
+Inside each example folder, you find a ``README.md`` file with detailed explanation and instrutions on how to run the experiment.
+First, you need to run the python script file named ``generate_input_files.py`` to generate the :ref:`input files<inputfiles>` needed by Mandyoc.
+Then, you may execute mandyoc directly from a terminal command or update the bash script ``run.sh`` accordingly to your setup and execute it to run the experiment.
+
```
Comment (aguspesce) [inline docs/files/benchmarks.rst:17]:
```suggestion

```

Comment (aguspesce) [reply docs/files/benchmarks.rst:17]: If we keep every README file in every examples' folder, this paragraph is not necessary

**Comment by aguspesce** [comment]
> So, I think we need to move 3.4. Examples to a dedicated section (it will be section 7 after 6 Benchmarks). Then update Benchmarks to reflect the new structure.
> 
> You suggested to
> 
>     create a README file, which explains how to run and create the models in each models folder.
> 
> which I think is covered by your PR #25 (for example this README.md)

@rafaelmds, you are right! If we maintain the README files for the examples, it is not necessary to add more information.

File: docs/files/benchmarks.rst [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 
+The scripts to build and run these numerical experiments are located in the `examples <https://github.com/ggciag/mandyoc/tree/main/examples>`_ folder of the Mandyoc repository.
+
+.. how-to-run-examples:
+
+**How to run examples:**
+
+Inside each example folder, you find a ``README.md`` file with detailed explanation and instrutions on how to run the experiment.
+First, you need to run the python script file named ``generate_input_files.py`` to generate the :ref:`input files<inputfiles>` needed by Mandyoc.
+Then, you may execute mandyoc directly from a terminal command or update the bash script ``run.sh`` accordingly to your setup and execute it to run the experiment.
```
Comment (aguspesce) [inline docs/files/benchmarks.rst:16]:
```suggestion
Inside each example folder, you find a ``README.md`` file with detailed explanation and instrutions on how to run the experiment.
```

Comment (rafaelmds) [reply docs/files/benchmarks.rst:16]: I think its better this way!

**Pull request merged**